Sources/PrimerSDK/Classes/User Interface/Components/PrimerFormView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  PrimerFormView.swift
+//  PrimerSDK
+//
+//  Created by Dario Carlomagno on 09/06/22.
+//
+
+import Foundation
```

Comment:
You need `#if canImport(UIKit)` and also `import UIKit` since you're dealing with views.